README.md

```diff
@@ -35,6 +35,22 @@ Do you want to know how your shipment is going and how long it will arrive? You
 ![javascript][]
 ![react][]
 
+## Pre requirements
+
+Before to start you need have installed the following programs:
+
+1. [Node.js](https://nodejs.org/) > v14.0.0
+
+   Node.js is a runtime environment for Javascript that allows us to use the language outside of the browser, in this case we use it to configure and run the development server.
+
+2. [npm](https://www.npmjs.com/) > v6.0.0
+
+   npm is a package manager for Node.js and comes installed with it, we use it to install and manage application dependencies.
```

Comment:
Maybe both can be on the same item, as one include the other.

Reply:
But npm is a different program

Reply:
Even so is nice to have both descriptions